app/views/orders/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1>ORDER</h1>
+
+<table>
+  <tr>
+    <th>Dish</th>
+    <th>Quantity</th>
+    <th>Price</th>
+  </tr>
+
+  <% @order.shopping_cart_items.each do |order_item| %>
+    <tr id="<%= "order_row_#{order_item.item.id}" %>">
+      <td><%= order_item.item.name %></td>
+      <td><%= order_item.quantity %></td>
+      <td style="white-space: nowrap;"><%= order_item.quantity * order_item.price %></td>
+    </tr>
+  <% end %>
+</table>
+
+<div>Total price: <%= @order.total%></div>
```

Comment:
Shouln't this be an #show page?

Reply:
There is no index page that should show all shopping carts. I want a clean URL so I'm using the index page as a show page. The id is handled in the session. 
Maybe I should add it something like this instead:
match '/order' => 'orders#show'

Reply:
Okay. I see the point